app/javascript/pages/browse/Filters.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section>
     <h2 class="title">Filters</h2>
-    <b-collapse v-for="(type, index) of filterTypes" :key="index" :open="true">
+    <b-collapse v-for="(type, index) of filterTypes" :key="index" :open="initialOpenStatus(index)">
```

Comment:
i don't see where we set this. on my dev, only the first filter section is open by default.